### ISSUE TITLE
Incompatibility between python-3.9 build-ipc and non privileged docker => luet build broken

### ISSUE THREAD

**Issue by welele08** [mocaccinoOS/desktop#217]
I tried building locally layers/net-tools-0.2+12. (git clone, checkout v0.4.3, luet build --image-repository quay.io/mocaccinocache/desktop --pull layers/net-tools).

it fail with a collection of  'File not found: /usr/lib/portage/python3.9/ebuild-ipc' from pre-merge check.

I suspect a side effect of python-3.9 introduction.  net-tools wasn't revbump and recomputed since that introduction. It will impact further dev cycle.

**Comment by joostruis** [comment]
As far as I know you always see this and you need to pass FEATURES="-sandbox -usersandbox -ipc-sandbox -pid-sandbox -network-sandbox"

**Comment by welele08** [comment]
FEATURES arg to luet command?

**Comment by welele08** [comment]
I've made further experiment. Since system-x use python-3.9, luet build command don't work to build package.

Way to experiment : 
- `git clone https://github.com/mocaccinoOS/desktop.git`
- `cd desktop`
- edit `packages/layers/qt/definition.yaml` to +1 version
- `luet --image-repository quay.io/mocaccinocache/desktop build --privileged --pull --only-target-package --live-output --no-spinner layers/qt`

Luet will try to build layers/qt, and emerge failed with "File not found: /usr/lib/portage/python3.9/ebuild-ipc" during pre-merge check. I tried this on 3 computer (two sabayon, one Debian), with luet-0.17.8.

I tried launching the builder image (`docker run -ti quay.io/mocaccinocache/desktop:builder-??? bash`).
A `emerge qt-core` failed with the same "File not found: /usr/lib/portage/python3.9/ebuild-ipc" error during pre-merge check.
I tried with a privileged container (`docker run --privileged -ti quay.io/mocaccinocache/desktop:builder-??? bash`).
In privileged container, `emerge qt-core` worked.

It seems building with "/python3.9/ebuild-ipc" need privileged container, which is not compatible with the `docker build <dockerfile>` command from luet build.

Maybe luet k8s-build-scheduler have a different behavior? But at the moment, `luet build `command seems incompatible with current desktop repo.
